addons/l10n_mx/views/report_templates.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="l10n_mx_external_layout_standard" inherit_id="web.external_layout_standard">
```

Comment:
I see that you correctly inherited the standard and background layouts, but what about boxed and clean ?

Btw, be careful with the forward-port as these layouts have changed names (boxed, striped, bold and standard) :-)

Reply:
I see the VAT in neither of these tho:
- https://github.com/odoo/odoo/blob/14.0/addons/web/views/report_templates.xml#L347-L348 (which you are correcting)
- https://github.com/odoo/odoo/blob/14.0/addons/web/views/report_templates.xml#L391-L392 (which you are not correcting)

I also don't understand your reply https://github.com/odoo/odoo/pull/129048#discussion_r1269531986. Could you elaborate on why:
- you do not use `== MX`
- you do not use `position="attributes"` instead of `replace`
?

I guess I'm missing something here :thinking: